packages/mobile/src/exchange/WithdrawCeloScreen.tsx

```diff
@@ -27,15 +29,18 @@ import { StackParamList } from 'src/navigator/types'
 import useSelector from 'src/redux/useSelector'
 import { useDailyTransferLimitValidator } from 'src/send/utils'
 import DisconnectBanner from 'src/shared/DisconnectBanner'
+import { divideByWei } from 'src/utils/formatting'
 
 type Props = StackScreenProps<StackParamList, Screens.WithdrawCeloScreen>
 
+const RANDOM_ADDRESS = '0xDCE9762d6C1fe89FF4f3857832131Ca18eE15C66'
+
 function WithdrawCeloScreen({ navigation }: Props) {
   const [accountAddress, setAccountAddress] = useState('')
   const [celoInput, setCeloToTransfer] = useState('')
 
   const goldBalance = useSelector((state) => state.goldToken.balance)
-  const goldBalanceNumber = new BigNumber(goldBalance || 0)
+  const goldBalanceNumber = useMemo(() => new BigNumber(goldBalance || 0), [goldBalance])
```

Comment:
Yeah, great catch, I'd like not to have it either, the problem is that we're creating a new `BigNumber` instance on each render and we pass this to the `useSendFee` hook, which uses it to see whether dependencies changed or not. If we pass different instances it assumes it changed and re-calculates the fee, which is undesired. The `useMemo` caches the instance of `BigNumber` so we use always the same unless the total balance changes for some reason.

Reply:
Oh, that's an interesting problem and a creative solution. 

I don't feel strongly about this but an alternative is to pass an `int` value instead and convert to `BigNumber` within the `useSendFee` function. If this would require a bunch of other changes, I'd understand if you'd rather not do it. But if that's the case, please leave a "to-do" so we fix this later.

As a side note, do you think we are having this issue in other places we are doing fee estimates?

Reply:
Missed this comment somehow. I changed the `amount` to a `string` since I saw it was being transformed to string right after anyways